Tests/PodToBUILDTests/UserConfigurableTests.swift

```diff
@@ -82,6 +82,15 @@ class UserConfigurableTests: XCTestCase {
         XCTAssertEqual(outputCopts?[1], "CoreGraphics")
         XCTAssertEqual(outputCopts?[2], "Foundation")
     }
+
+    func testUserOptionPresevesDotExtensions() {
+        let target = TestTarget()
+        let attributes = UserConfigurableTargetAttributes(keyPathOperators:  ["TestTarget.copts += TargetConditionals.h"])
```

Comment:
PR seems good. Totally orthogonal, but would we add a `-DTARGET_OS_IPHONE` to workaround the issue in bugsnag?

Reply:
it could slam it in, i suppose. fixing this will help too.